aggregator_api/src/lib.rs

```diff
@@ -409,6 +421,22 @@ mod models {
     use serde::{Deserialize, Serialize};
     use url::Url;
 
+    #[allow(dead_code)] // allowed in order to fully describe the interface and for later use
+    #[derive(Serialize, PartialEq, Eq, Debug)]
+    pub(crate) enum AggregatorRole {
+        Either,
+        Leader,
+        Helper,
+    }
+
+    #[derive(Serialize, PartialEq, Eq, Debug)]
+    pub(crate) struct AggregatorApiConfig {
```

Comment:
Pedantic, take it or leave it: This is a slight misnomer, since it's not describing the aggregator API. I'd also argue it's not describing a configuration either--I would suggest `AggregatorCapabilities`.

Reply:
I started out calling it capabilities but decided its "how the aggregator api is configured" from the perspective of the api consumer (divviup-api). Specifically, the public dap url isn't really a capability, but it is a configuration. Other properties of this struct are both capabilities and also configuration